docs/en/sql-reference/functions/string-functions.md

```diff
@@ -2532,3 +2532,37 @@ Result:
 │ foo                        │
 └────────────────────────────┘
 ```
+## strncmp
+
+Compare two strings.
+
+**Syntax**
+```sql
+strncmp(a_str, a_offset, b_str, b_offset, n);
+```
+
+**Arguments**
+- `a_str` — The first string to compare.
+- `a_offset` — The starting position (zero-based index) in `a_str` from which the comparison begins.
+- `b_str` — The second string to compare.
+- `b_offset` — The starting position (zero-based index) in `b_str` from which the comparison begins.
+- `n` — The number of bytes to compare in both strings, starting from their respective offsets.
+
+**Returned value**
+- -1 — If `a_str`[`a_offset`: `a_offset` + `n`] < `b_str`[`b_offset`:`b_offset` + `n`].
```

Comment:
Somewhere, we'll need to specify what should happen if
- an offset is bigger than the length of an input string,
- `n` exceeds the end of the input string (starting from the offset)